src/runs.tsx

```diff
@@ -23,34 +41,49 @@ type RouteParams = { id: string };
 
 interface Props extends RouteComponentProps<RouteParams> { }
 
-class RunsPage extends React.Component<Props, IState> {
-    public constructor(props: Props) {
+class RunsPage extends React.Component<Props & IProps, IState> {
+    public constructor(props: Props & IProps) {
         super(props);
         this.state = {
             catalogs: []
         };
     }
 
     render() {
+        const { classes } = this.props;
         return (
-          <Container maxWidth="sm">
-          <Box my={4}>
-            <Typography variant="h4" component="h1" gutterBottom>
-              This page lists runs for {this.props.match.params.id}
-              </Typography>
-              <Typography variant="h6" component="h1" gutterBottom>
-              <div>
-                <ul>
-                    {this.state.catalogs.map(catalog => (
-                        <li key={catalog.uid}>
-                            <Link color="inherit" component={RouterLink} to={"/runs/" + this.props.match.params.id + "/" + catalog.uid}>{catalog.uid}</Link>
-                        </li>
-                    ))}
-                </ul>
-              </div>
-            </Typography>
+          <div style={{ width: '100%' }}>
+              Showing {this.props.match.params.id}
+          <Box display="flex" flexWrap="wrap" p={1} m={1} bgcolor="background.paper">
+          {this.state.catalogs.map(catalog => (
+          <Card className={classes.root}>
+            <CardActionArea>
+              <CardMedia
+                className={classes.media}
+                image={"/previews/" + catalog.uid}
```

Comment:
As discussed at the podman call, I think it should have `/db/` in the route.
```suggestion
                image={"/db/previews/" + catalog.uid}
```